articles/quickstart/spa/angular2/05-token-renewal.md

```diff
@@ -69,15 +76,17 @@ console.log('Listening on http://localhost:3001');
 </html>
 ```
 
-In this example, the server is running at `localhost:3001` and this value is hardcoded as the `redirectUri` method below. The `silent.html` file makes reference to `localhost:4200` which is the address that the Angular CLI uses for development servers.
+::: note
+In this example, the server is running at `localhost:3001`. This value is hardcoded as the `redirectUri` method. The `silent.html` file refers to `localhost:4200` which is the address that the Angular CLI uses for development servers.
+:::
 
 ::: note
-Be sure to add `http://localhost:3001/silent` to the **Callback URLs** section in your application's client settings.
+Add `http://localhost:3001/silent` to the **Callback URLs** section in your application's client settings.
```

Comment:
Actually this is not a section. It is a field in the Client Settings:

go to Dashboard > Clients > select a client > Settings > the field is named `Allowed Callback URLs`

![image](https://user-images.githubusercontent.com/11715799/29560099-a8e13140-8739-11e7-895a-5ac0c8efa3ac.png)